### ISSUE TITLE
A solution to reference image in CSS

### ISSUE THREAD

**Issue by lihan** [FormidableLabs/radium#372]
With CSS file, you could reference images using relative path, it will use the relative path to the CSS file location.

With in JS file, this is not the case. If you reference an image using relative path, the path will be relative to displayed page which won't work with image files.

Will there be a solution to resolve this?

**Comment by alexlande** [comment]
This isn't Radium-specific, but you have a few options: 

- Use absolute paths (I would recommend this in general)
- Use a build system that will handle your image paths for you, like Webpack (this would convert them to absolute paths under the hood, anyway)

**Comment by alexgorbatchev** [comment]
I think this topic deserves attention in the docs (at least in the FAQs). Most, if not all, build systems are made to handle static assets through CSS files. Trying to figure out how to do this when there are no .css files isn't a trivial task for a beginner. A few pointers in the right direction might be a great start.

**Comment by ianobermiller** [comment]
Fair enough. Have you figured out anything so far that would be useful to add?
